rasa/nlu/test.py

```diff
@@ -365,7 +365,7 @@ def plot_entity_confidences(
         if prediction not in (NO_ENTITY, target)
     ]
 
-    plot_utils.plot_histogram([pos_hist, neg_hist], title, hist_filename)
+    plot_utils.plot_comparative_histogram([pos_hist, neg_hist], title, hist_filename)
```

Comment:
Please preserve the original method name (`plot_histogram`) for backwards compatibility

Reply:
Done